msteams-platform/concepts/design/design-teams-app-overview.md

```diff
@@ -134,6 +134,14 @@ Understand how people add, use, and manage Teams apps to make the most of each c
 * [Task modules](../../task-modules-and-cards/task-modules/design-teams-task-modules.md)
 * [Adaptive Cards](../../task-modules-and-cards/cards/design-effective-cards.md)
 
+## App customization
+
+Understand how the Teams admin can customize or rebrand the app based on the organization's need. This customization is enabled if you define the `configurableProperties` in the manifest schema. For more information, see [Customize apps in Microsoft Teams](/MicrosoftTeams/customize-apps).
```

Comment:
Could you please add reference to this in [Apps in public developer preview](https://review.docs.microsoft.com/en-us/microsoftteams/platform/resources/dev-preview/developer-preview-features?branch=pr-en-us-2964) and [What's new](https://review.docs.microsoft.com/en-us/microsoftteams/platform/whats-new?branch=pr-en-us-2964) section?